src/components/listitems/ListItemNav.tsx

```diff
@@ -41,20 +43,27 @@ const legacyStyles = StyleSheet.create({
   }
 });
 
-export type ListItemNav = WithTestID<{
+export type ListItemNavPartialProps = WithTestID<{
```

Comment:
Maybe it's better to not export the `PartialProps` type:
```suggestion
type ListItemNavPartialProps = WithTestID<{
```

Reply:
Addressed into 48ff0bbd8c7d54c311814065e18c302e6c61a6a6